.travis.yml

```diff
@@ -22,6 +22,7 @@ env:
   global:
     - CI="travis"
     - JRUBY_OPTS="--server -J-Xms512m -J-Xmx1024m"
+    - PATH="${PWD}/mongodb-linux-x86_64-${MONGODB}/bin:$PATH"
```

Comment:
I guess this is not required any more once you remove the helpers manually launching mongod